app.py

```diff
@@ -11,6 +11,8 @@
 from prof_dashboard import RemoveTeam
 from prof_dashboard import SetDate
 from report import TeamReportListView, GeneratedProfessorReportView, GeneratedAnonymousReportView
+from viewStudent import ViewStudent
```

Comment:
nit: module/file names should be snake_case (meaning view_student rather than viewStudent)